FESS-GUI/mainwindow.cpp

```diff
@@ -288,19 +293,40 @@ void MainWindow::on_configButton_clicked()
     ui->stackedWidget->setCurrentIndex(1);
 }
 
-void MainWindow::on_velocitySlider_valueChanged(int value)
+void MainWindow::on_velocitySlider_valueChanged(int velocity)
```

Comment:
I think it makes more sense to change the expected values when the "go" button is pressed. We can talk about it in the meeting today.